src/static/js/modules/loan-comparison/common.js

```diff
@@ -153,7 +153,27 @@ common.errorMessages = {
 common.propLabels = {
     'downpayment': 'Down payment',
     'points':      'Discount point and credits',
-    'arm-type':    'ARM type'
+    'arm-type':    'ARM type',
+    'price': 'House price',
+    'third-party-services': 'Third party services',
+    'closing-costs': 'Projected cash to close',
+    'monthly-payment': 'Projected monthly payment',
+    'overall-costs': 'Projected overall costs',
+    'discount': 'Discount points or credits',
+    'processing': 'Origination and processing fees',
+    'lender-fees': 'Lender fees',
+    'third-party-services': 'Third party services',
+    'insurance': 'Mortgage insurance', 
+    'taxes-gov-fees': 'Taxes and government fees', 
+    'prepaid-expenses': 'Prepaid expenses',
+    'initial-escrow': 'Initial escrow deposit',
+    'monthly-taxes-insurance': 'Monthly taxes and insurance',
+    'monthly-hoa-dues': 'Monthly HOA dues', 
+    'monthly-principal-interest': "Monthly principal and interest",
+    'monthly-mortgage-insurance': 'Monthly mortgage insurance',
+    'loan-term': 'Loan term',
+    'principal-paid': 'Principal paid',
+    'interest-fees-paid': 'Interest and fees paid'
 };
 
```

Comment:
We could probably just have entries here where the labels don't match the property names. In the inputs table, I'm generating labels where possible -- we can move that code into a function here for reuse.